Modix.Bot/Modules/PromotionsModule.cs

```diff
@@ -69,15 +69,58 @@ public async Task Campaigns()
 
         [Command("nominate")]
         [Summary("Nominate the given user for promotion")]
-        public Task Nominate(
+        public async Task Nominate(
             [Summary("The user to nominate")]
                 IGuildUser subject,
-            [Summary("The role for the user to be promoted to")]
-                IRole targetRole,
             [Remainder]
             [Summary("A comment to be attached to the new campaign")]
                 string comment)
-            => PromotionsService.CreateCampaignAsync(subject.Id, targetRole.Id, comment);
+        {
+            await PromotionsService.CreateCampaignAsync(subject.Id, comment, c => Confirm(c));
+
+            async Task<bool> Confirm(ProposedPromotionCampaignBrief proposedPromotionCampaign)
```

Comment:
This is actually the first time I've ever seen a local function, out in the wild. Three cheers!